features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt

```diff
@@ -135,6 +174,108 @@ fun RoomMemberListTopBar(
     )
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun RoomMemberSearchBar(
+    query: String,
+    state: RoomMemberSearchResultState,
+    active: Boolean,
+    modifier: Modifier = Modifier,
+    placeHolderTitle: String = stringResource(io.element.android.libraries.ui.strings.R.string.common_search_for_someone),
```

Comment:
We usually add `import io.element.android.libraries.ui.strings.R as StringR` and then just `StringR.string.xxxx`.

Reply:
Done, though it will be hard to remember this by heart, is there any lint check for this maybe?

Reply:
I don't think there is. We could create one, or maybe you could add a live template for this?

<img width="709" alt="image" src="https://github.com/vector-im/element-x-android/assets/480955/036261be-22fb-4b38-9953-4ba9012e7e6b">